### ISSUE TITLE
Publish Report on State of Knowledge for Web3 Governance

### ISSUE THREAD

**Issue by dmc314** [protocol/CryptoEconLab#43]

**Comment by dmc314** [comment]
This report is a low priority at the moment. There is a discussion pending on Filecoin governance reforms for core devs, emergency fips, and/or cryptoeconomic fips, the outcome of which may inform whether related research will be necessary in January.

There is also the possibility that we would wrap this governance benchmarking and SoK research into a larger project and offer it as a bundled deal to multiple DAO and Blockchain clients in 2023. If we can get a dozen interested clients to pitch in some amount of compensation that will help them each make better governance reforms, this could be a valuable project.

We plan to start drafting some scaffolds of the research, but revisit in earnest in January. 

FYI @emceecoy, @juanpmcianci , @zixuanzh
